specification/managedservices/resource-manager/Microsoft.ManagedServices/preview/2020-02-01-preview/managedservices.json

```diff
@@ -1044,24 +1044,24 @@
         "roleDefinitionId"
       ],
       "type": "object",
-      "description": "Authorization tuple containing principal Id (of user/service principal/security group) and role definition id."
+      "description": "The user ID and permanent role for a principal in the managedBy tenant."
     },
     "EligibleAuthorization": {
       "properties": {
         "principalId": {
           "type": "string",
-          "description": "Principal Id of the security group/service principal/user that would be delegated permissions to the projected subscription"
+          "description": "The identifier of the Azure Active Directory principal."
         },
         "principalIdDisplayName": {
           "type": "string",
-          "description": "Display name of the principal Id."
+          "description": "The display name of the principal."
```

Comment:
>principal [](http://example.com/codeflow?start=50&length=9)

AAD principal

Reply:
Changed most references of "principal" to "Azure Active Directory principal"